src/foam/nanos/controller/ApplicationController.js

```diff
@@ -240,6 +248,8 @@ foam.CLASS({
     },
 
     function initE() {
+      window.addEventListener('resize', () => this.displayWidth = window.innerWidth);
```

Comment:
Do this instead:
```
window.addEventListener('resize', () => this.displayWidth = undefined);
```
This will clear the property which will cause your factory to re-run so you don't need to duplicate the logic of that factory here.

Reply:
Done.